tests/virt_autotest/login_console.pm

```diff
@@ -35,6 +35,18 @@ sub ipmitool {
     return $stdout;
 }
 
+sub set_ssh_console_timeout_before_use {
+    reset_consoles;
+    select_console('root-console');
+    set_ssh_console_timeout('/etc/ssh/sshd_config', '28800');
+    reset_consoles;
+    select_console 'sol', await_console => 0;
+    send_key 'ret';
+    check_screen([qw(linux-login virttest-displaymanager)], 60);
+    save_screenshot;
+    send_key 'ret';
+}
```

Comment:
This function does more than what its name tells. Can you adjust the name a little bit?

Reply:
set_ssh_console_timeout_before_use ? No, its name just reflects its main purpose